Assets/SEE/XR/XRSEEActions.cs

```diff
@@ -0,0 +1,218 @@
+    /// <summary>
+    /// Is true, when the button for the primary actions is pressed.
+    /// </summary>
+    public static bool Selected { get; set; }
+
+    public static bool SelectedFlag { get; set; }
+    /// <summary>
+    /// The GameObject, which should be rotated.
+    /// </summary>
+    public static GameObject RotateObject { get; set; }
+    /// <summary>
+    /// This method gets called, when the button for the primary actions is pressed.
+    /// </summary>
+    /// <param name="context">Information provided to action callbacks about what triggered an action.</param>
+    private void Action(InputAction.CallbackContext context)
+    {
+        if (hovering)
+        {
+            if (GlobalActionHistory.Current() == ActionStateTypes.Move && Selected == true)
+            {
+                Selected = false;
+                SelectedFlag = false;
+            }
+            else
+            {
+                if (GlobalActionHistory.Current() == ActionStateTypes.Rotate)
+                {
+                    RayInteractor.TryGetCurrent3DRaycastHit(out RaycastHit hit);
+                    RotateObject = hit.collider.transform.gameObject;
+                    Selected = true;
+                }
+                Selected = true;
+                if (GlobalActionHistory.Current() != ActionStateTypes.Move)
+                {
+                    SelectedFlag = true;
+                }
+            }
+        }
+        else
+        {
+            InteractableObject.ReplaceSelection(null, true);
+        }
+    }
+    /// <summary>
+    /// Will be true, when the TreeView is open, while the user tries to move a node.
+    /// </summary>
+    public static bool CloseTreeView { get; set; }
+    /// <summary>
+    /// Is true, when the Tooltip is getting activated.
+    /// </summary>
+    public static bool TooltipToggle { get; set; }
+    /// <summary>
+    /// Is true, while the Tooltip is open.
+    /// This bool is used, to close the Tooltip, if the user does not want
+    /// to use an action.
+    /// </summary>
+    public static bool OnSelectToggle { get; set; }
+    /// <summary>
+    /// Is true, when the user opens the tooltip in the treeview.
+    /// </summary>
+    public static bool OnTreeViewToggle { get; set; }
+    /// <summary>
+    /// The treview-entry for which the tooltip should be shown.
+    /// </summary>
+    public static GameObject TreeViewEntry { get; set; }
+    /// <summary>
+    /// This method gets called, when the button for the tooltip is pressed.
+    /// </summary>
+    /// <param name="context">Information provided to action callbacks about what triggered an action.</param>
+    private void Tooltip(InputAction.CallbackContext context)
+    {
+        if (OnTreeViewToggle)
+        {
+            TooltipToggle = true;
+            TreeViewEntry.TryGetComponentOrLog(out PointerHelper pointerHelper);
+            pointerHelper.ThumbstickEvent.Invoke(new PointerEventData(EventSystem.current));
```

Comment:
It seems like the `PointerHelper` must be present, so `TryGetComponent` doesn't seem like the best fit here:
```suggestion
            TreeViewEntry.MustGetComponent<PointerHelper>().ThumbstickEvent.Invoke(new PointerEventData(EventSystem.current));
```